x-pack/plugins/siem/public/containers/hosts/index.tsx

```diff
@@ -34,6 +33,8 @@ export interface HostsArgs {
 export interface OwnProps extends QueryTemplateProps {
   children: (args: HostsArgs) => React.ReactNode;
   type: hostsModel.HostsType;
+  startDate: number;
+  endDate: number;
```

Comment:
Should these be of type `Date`?

Reply:
To make this happen, we need to finish our scalar to update input to take isoString and convert to epoch for ES. It will happen but now here ;)